server/src/internalClusterTest/java/org/opensearch/search/suggest/CompletionSuggestSearchIT.java

```diff
@@ -1180,8 +1182,10 @@ public void testSkipDuplicates() throws Exception {
             int weight = randomIntBetween(0, 100);
             weights[id] = Math.max(weight, weights[id]);
             String suggestion = "suggestion-" + String.format(Locale.ENGLISH, "%03d", id);
+            logger.info("Creating {}, id {}, weight {}", suggestion, i, id, weight);
             indexRequestBuilders.add(
                 client().prepareIndex(INDEX)
+                    .setRefreshPolicy(WAIT_UNTIL)
```

Comment:
another nit - you could also use the immediate policy here that performs a blocking refresh instead of waiting for the next scheduled refresh to occur, which in this case would be the default (1s).

Reply:
Since there can be up to 100 documents, I'd think it makes sense for them to all wait at once.  However, it sounds like this standard should be applied in all tests cases - would you want to make a follow up issue?

Since you used the term `nit` I'm going to resolve this without confirmation - thanks.

Reply:
There is really no standard here on refresh policy, its dependent on how the test is performing writes and what its aimed at testing.

Though I was wrong here and wait_until is more appropriate for this test vs forcing 100x refreshes.  The test is building up to 100 separate write requests each with a single doc, and indexRandom executes them async.

I'm also curious as @jed326 called out why refresh is the issue here given indexRandom's forced refresh isn't fixing this here as that happens [after](https://github.com/opensearch-project/OpenSearch/blob/bc74731a537c461cd1d7666bc3be82fbe18e143d/test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java#L1686) all writes are ack'd.